src/pages/find/components/vehicle_item.vue

```
@@ -0,0 +1,60 @@
+                        品牌
+                    </text>
+                    <text class="value">
+                        {{ item?.brandName || '--' }}
+                    </text>
+                </view>
+                <view class="info-item">
+                    <text class="title">
+                        车型
+                    </text>
+                    <text class="value">
+                        {{ sizeEnum[item?.sizeType] || '--' }}
```

ESLint

[error] Expected indentation of 12 spaces but found 24 spaces.

(vue/html-indent)